src/index.js

```diff
@@ -19,7 +19,7 @@ if ( currentLastTimestamp ){
 
   // If the current last timestamp from cookies / process.env does not match the stored one, it means user has run init or has built the app again
   // Hence, clear the whole persisted store.
-  if ( currentLastTimestamp !== storedLastTimestamp ){
+  if ( currentLastTimestamp !== storedLastTimestamp && storedLastTimestamp < currentLastTimestamp){
```

Comment:
I think the first statement boolean value will always be equal to the second statement.

Reply:
Not always; I have found out that storedLastTimestamp is sometimes greater than currentLastTimestamp

Reply:
Ah right... What I actually want to say that, I think that the first statement is redundant

Reply:
You are right; I have removed the first statement. Many thanks